Library/Homebrew/.bundle/config

```diff
@@ -1,6 +1,6 @@
 ---
-BUNDLE_BIN: "../bin"
-BUNDLE_PATH: "../vendor/bundle"
+BUNDLE_BIN: "false"
 BUNDLE_DISABLE_SHARED_GEMS: "true"
 BUNDLE_JOBS: "4"
```

Comment:
Is this automatically generated? Seems weird to have this fixed at 4.

Reply:
It's not automatically generated but unfortunately whatever you set it to at runtime ends up getting written to this configuration file so we want to set it to a sensible default that's not `1` (to speed things up a bit).